projects/plugins/jetpack/changelog/add-9.8-alpha

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: other
+
+bump ver number
```

Comment:
No need for a changelog entry for this, IMO.
```suggestion
Comment: bump ver number

```

Reply:
Did the CLI fast and didn't read the prompt I guess!